--- pigz-2.7/pigz.c~	2022-01-15 19:18:20.000000000 +0000
+++ pigz-2.7/pigz.c	2022-01-16 22:56:12.331043423 +0000
@@ -4343,10 +4343,10 @@
     g.procs = nprocs(8);
     g.shift = x2nmodp(g.block, 3);
 #endif
-    g.rsync = 0;                    // don't do rsync blocking
+    g.rsync = 1;                    // don't do rsync blocking
     g.setdict = 1;                  // initialize dictionary each thread
     g.verbosity = 1;                // normal message level
-    g.headis = 3;                   // store name and time (low bits == 11),
+    g.headis = 1;                   // store name and time (low bits == 11),
                                     // restore neither (next bits == 00),
                                     // where 01 is name and 10 is time
     g.pipeout = 0;                  // don't force output to stdout
